recipes/av/meta.yaml

```diff
@@ -22,11 +22,12 @@ requirements:
     - setuptools
     - cython
     - pkg-config
-    - ffmpeg
+    - ffmpeg 2.8.*
   run:
     - python
-    - ffmpeg
+    - ffmpeg 2.8.*
     - pillow
+    - numpy x.x
```

Comment:
I think if it isn't a build dependency, you can get away just `numpy` and not `numpy x.x`. This dramatically reduces the number of package specializations that get made, since there currently np 1.9, 1.10, and 1.11 are all supported.